### DISCUSSION TITLE
Pedestrians

### DISCUSSION THREAD

**Discussion by fabio-r-goncalves** [eclipse/mosaic#257] [Q&A] [answered]
I am trying to import a sumo scenario with pedestrian routes. Is there a way to have pedestrian routes and pedestrians in mosaic?
Thanks.

**Comment by fabio-r-goncalves** [comment]
Ok, so I think one of the problems is related to scenario convert, as it does not import routes with pedestrian paths. Is there any way to import pedestrian routes?

**Reply by schwepmo** [reply, marked as answer]
Hey,
currently there doesn't exist any representation of pedestrians in MOSAIC, so extending scenario-convert won't allow to dynamically adjust the behaviour of pedestrians. However, if you just want pedestrians to act as "background traffic" you should be able to follow [this tutorial](https://www.eclipse.org/mosaic/tutorials/integrate_existing_sumo_scenarios/) to configure pedestrians using the SUMO route-file.

**Reply by fabio-r-goncalves** [reply]
Thanks, that was exactly what I was looking for.
